enf-reducer/rte-helpers.metta

```diff
@@ -75,6 +75,20 @@
         $current
     )
 )
+
+(= (foldLeftBinary $f $acc $current Nil) $acc)
+(= (foldLeftBinary $f $acc $current (Cons $x $xs)) 
```

Comment:
And I noticed you didn't use the `$acc` in your code in the `foldLeftBinary` function? If you don't use it, remove it from the constraint :).